Resources/public/js/views/services/plugins/ez-userdraftsplugin.js

```diff
@@ -0,0 +1,245 @@
+        },
+
+        /**
+         * Collects drafts data
+         *
+         * @method _collectDraftsData
+         * @protected
+         * @param error {Boolean} is error?
+         * @param versions {Array} list of versions
+         * @return {Y.Promise}
+         */
+        _collectDraftsData: function (error, versions) {
+            var host = this.get('host');
+
+            if (error) {
+                host._error(error.message);
+                this._target.set('loadingError', true);
+                this._resetTempVariables();
+
+                return;
+            }
+
+            versions.sort(this._sortByModificationDate);
+            versions.length = this._targetLimit;
+
+            Y.Promise
+                .resolve(versions)
+                .then(Y.bind(this._loadDraftContentInfo, this))
+                .then(Y.bind(this._loadDraftContentType, this))
+                .then(Y.bind(this._fillDraftContentTypeModel, this))
+                .catch(Y.bind(host._error, host));
+        },
+
+        /**
+         * Gets content info for each version
+         *
+         * @method _collectDraftsData
+         * @protected
+         * @param versions {Array} list of versions
+         * @return {Y.Promise}
+         */
+        _loadDraftContentInfo: function (versions) {
+            var drafts = this.get('drafts'),
+                contentService = this.get('host').get('capi').getContentService(),
+                promises = [];
+
+            versions.forEach(function (version) {
+                var contentId = version.get('resources.Content');
+
+                drafts.add(version);
+
+                promises.push(new Y.Promise(Y.bind(function (resolve, reject) {
+                    contentService.loadContentInfo(contentId, Y.bind(function (error, response) {
+                        if (error) {
+                            reject('Cannot load content info: ' + contentId);
+                            this._target.set('loadingError', true);
+                            this._resetTempVariables();
+
+                            return;
+                        }
+
+                        resolve(response.document.Content);
+                    }, this));
+                }, this)));
+            });
+
+            return Y.Promise.all(promises);
+        },
+
+        /**
+         * Loads content types for each content version
+         *
+         * @method _loadDraftContentType
+         * @protected
+         * @param data {Array} list of content
+         * @return {Y.Promise}
+         */
+        _loadDraftContentType: function (data) {
+            var contentTypeService = this.get('host').get('capi').getContentTypeService(),
+                drafts = this.get('drafts'),
+                promises = [];
+
+            data.forEach(Y.bind(function (contentInfo, index) {
+                drafts.item(index).set('contentInfo', contentInfo);
```

Comment:
Of course it doesn't have anything like this. Where should I store required data?